featurebyte/models/observation_table.py

```diff
@@ -74,6 +74,7 @@ class UploadedFileInput(FeatureByteBaseModel):
     """
 
     type: Literal[RequestInputType.UPLOADED_FILE]
+    file_name: str
```

Comment:
Should this be optional? Wondering if it would break retrieving or listing existing observation tables created before this field was added.

Reply:
true, will update.